### DISCUSSION TITLE
Most requested features

### DISCUSSION THREAD

**Discussion by slawkens** [slawkens/myaac#212] [Q&A]
Hello guys,

please write what are the most requested features that are missing currently in MyAAC.

If feature is already written, just upvote it (give 👍) and don't repeat yourself.

I will take all your propositions into consideration when developing the next version.

Let's start!

**Comment by slawkens** [comment]
Guild Wars

**Comment by slawkens** [comment]
i18 (Multi-Language)

**Comment by slawkens** [comment]
two factor authentication for TFS 1.x

**Reply by slawkens** [reply]
Done in: https://github.com/slawkens/myaac/pull/327

**Comment by slawkens** [comment]
better gallery with carousel

**Reply by slawkens** [reply]
Done in develop branch.

**Comment by chakoo2002** [comment]
converting znote and modern ACC templates to MyACC

**Comment by slawkens** [comment]
Private Messages System

**Comment by Rdz92** [comment]
Leave house and sell house function via AAC/website sort of like real tibia does (sorry if it's already an feature)

**Comment by Berzekz** [comment]
- Coins/Premium sales statistic
- Payments page like real tibia
- Multiworld system like real tibia (Guilds/Online/Create Account/Highscores)
- Char Bazaar

**Comment by vitaaapng** [comment]
Powergamers (rank who make more exp per day)

**Comment by kokekanon** [comment]
[character page]
add OPTION (if) "season rank based on highscores" on
config
```
'characters' => array( // what things to display on character view page (true/false in each option)
      'ranking' => true, //https://otland.net/threads/create-season-rank-based-on-highscores.282083
```

1 click on admin panel
sql execution of
https://otland.net/threads/create-season-rank-based-on-highscores.282083/#post-2703996

![image](https://user-images.githubusercontent.com/114332266/222665276-3ade6115-7c80-4689-8473-645f122849e6.png)

**Comment by kokekanon** [comment]
[plugins]
Guild Shop

**Comment by kokekanon** [comment]
[Admin panel] dashboard statistics

Day of the week when there are more players

hours in the day that there is more player

so that myacc user can make decisions regarding events, reset, announcements

**Comment by kokekanon** [comment]
[plugins] Upcoming events
panel widget ->

![image](https://user-images.githubusercontent.com/114332266/222664949-d1e7550e-bc8c-4563-a5a5-179335a2ad31.png)


https://tibiaking.com/forums/topic/76526-widget-upcoming-events-para-tfs-12/

**Reply by slawkens** [reply]
Hi, just done this - https://gist.github.com/slawkens/ab98fad2cf0e9a4b101ed897191374c9

More info in the gist.
